google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingHandler.java

```diff
@@ -310,12 +354,15 @@ private LogEntry entryFor(LogRecord record) {
         .addLabel("levelName", level.getName())
         .addLabel("levelValue", String.valueOf(level.intValue()))
         .setSeverity(severityFor(level));
+
     enhanceLogEntry(builder, record);
     return builder.build();
   }
   
   protected void enhanceLogEntry(LogEntry.Builder builder, LogRecord record) {
```

Comment:
Make this private now, or better yet, just move the implementation to where it's called, because it doesn't need to be overridden anymore.